docs/using-mysql.md

```diff
@@ -2,6 +2,10 @@
 
 A MySQL database is recommended for improved performance and allowing you to have ticket transcripts/archives. SQLite is too slow to support storing messages.
 
+!!! danger "WARNING" 
+
+	For be sure to don't have Sequelize error on bot startup use the 'utf8_unicode_ci' collation for your database.
```

Comment:
Please check your English is proper too that'd be appreciated though we can help with this if you need us to 😄